dubbo-monitor/dubbo-monitor-default/src/test/java/org/apache/dubbo/monitor/dubbo/MetricsFilterTest.java

```diff
@@ -56,32 +59,32 @@
 import static org.apache.dubbo.monitor.Constants.DUBBO_PROVIDER_METHOD;
 import static org.apache.dubbo.monitor.Constants.METHOD;
 import static org.apache.dubbo.monitor.Constants.SERVICE;
+import static org.mockito.BDDMockito.given;
+import static org.mockito.Mockito.doNothing;
+import static org.mockito.Mockito.mock;
 
 public class MetricsFilterTest {
 
-    private final Invoker<DemoService> serviceInvoker = new Invoker<DemoService>() {
-        @Override
-        public Class<DemoService> getInterface() {
-            return DemoService.class;
-        }
+    private Invoker<DemoService> serviceInvoker;
 
-        public URL getUrl() {
-            return URL.valueOf("dubbo://" + NetUtils.getLocalHost() + ":20880/org.apache.dubbo.monitor.dubbo.service.DemoService");
-        }
+    @BeforeEach
```

Comment:
@ralf0131 In the first PR, I made these changes to make it easier to create new unit tests (is using Mockito).

Should I remove this modification or open a new PR?

Reply:
If it is un-related, it would be better to create a separate one.